README.md

```
@@ -25,6 +25,17 @@ echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee -a /etc/
 sudo apt-get update
 sudo apt-get install -y python3-pip apt-transport-https kubectl
 ```
+# 1. Download the latest version of kubectl
+curl -LO "https://dl.k8s.io/release/$(curl -Ls https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)